src/app/slider/slider-sample-1/slider-sample-1.component.css

```diff
@@ -0,0 +1,14 @@
+.container {
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-bottom: 30px;
+    margin-top: 30px;
+  }
```

Comment:
Could we use a shorthand for setting margin properties? Like:
margin: 30px 20px;